core/src/executor/evaluate/error.rs

```diff
@@ -7,6 +7,9 @@ use {
 
 #[derive(Error, Serialize, Debug, PartialEq)]
 pub enum EvaluateError {
+    #[error(transparent)]
+    ChronoFormat(#[from] ChronoFormatError),
```

Comment:
That's interesting that we have 2-level depth error conversion.
Looks good :) thanks!